src/test/java/bad/robot/excel/workbook/PoiWorkbookTest.java

```diff
@@ -149,4 +154,16 @@ public void replaceThenLoadBlankCell() throws IOException {
         assertThat(adaptPoi(getCellForCoordinate(coordinate(A, 1), modified.workbook())), is(instanceOf(BlankCell.class)));
     }
 
-}
\ No newline at end of file
+    @Test
+    public void shouldCountMergedCellsCorrectly() throws IOException, InvalidFormatException {
+        // Create a workbook with a single empty sheet that has merged cells.
+        XSSFWorkbook workbook = new XSSFWorkbook();
+        XSSFSheet sheet = workbook.createSheet("Sheet1");
+        int firstMergedCol = 1;
+        int lastMergedCol = 5;
+        sheet.addMergedRegion(new CellRangeAddress(0, 0, firstMergedCol, lastMergedCol));
+
+        // Compare the workbook to an empty workbook without merged cells.
+        assertThat(workbook, sameWorkbook(getWorkbookFromFactory("emptySheet.xlsx")));
```

Comment:
and I assume you really want something like

```java
assertThat(workbook, is(not(sameWorkbook(getWorkbook("emptySheet.xlsx")))));
```
and maybe `@Ignore` the test.

Interestingly, the output when this test fails is terrible.

```bash
Expected: is not entire workbook to be equal
     but: was <Name: /xl/workbook.xml - Content Type: application/vnd.openxmlformats-officedocument.spreadsheetml.sheet.main+xml>
Expected :is not entire workbook to be equal
     
Actual   :<Name: /xl/workbook.xml - Content Type: application/vnd.openxmlformats-officedocument.spreadsheetml.sheet.main+xml>
```

The matcher stuff is all about making this output easy to read, so we should do something about that also.